specs/phase0/p2p-interface.md

```diff
@@ -593,10 +592,10 @@ The responder MUST:
 If steps (1), (2), or (3) fail due to invalid, malformed, or inconsistent data, the responder MUST respond in error.
 Clients tracking peer reputation MAY record such failures, as well as unexpected events, e.g. early stream resets.
 
-The entire request should be read in no more than `RESP_TIMEOUT`.
-Upon a timeout, the responder SHOULD reset the stream.
+The responder MAY rate-limit chunks by withholding each chunk until capacity is available. The responding side MUST NOT respond with an error or close the stream when rate limiting.
```

Comment:
I'd always use `The responder` instead of `The responding side`.

Reply:
[a232b25](https://github.com/ethereum/consensus-specs/pull/3767/commits/a232b25e7f1067b1301be37c9daab9b8685ee4dc)